docs/usage_guide/advanced_usage/index.rst

```diff
@@ -31,7 +31,7 @@ Modin also supports these experimental APIs on top of pandas that are under acti
 - :py:func:`~modin.experimental.pandas.read_sql` -- add optional parameters for the database connection
 - :py:func:`~modin.experimental.pandas.read_custom_text` -- read custom text data from file
 - :py:func:`~modin.experimental.pandas.read_pickle_distributed`  -- read multiple files in a directory
-- :py:meth:`~modin.experimental.pandas.DataFrame.to_pickle_distributed` -- write to multiple files in a directory
+- :py:meth:`~modin.experimental.pandas.DataFrame.modin.to_pickle_distributed` -- write to multiple files in a directory
```

Comment:
Why when we click on this method in the docs, we navigate to the method with the signature modin.to_pickle_distributed but not DataFrame.modin.to_pickle_distributed?